packages/synapse-react-client/src/utils/SynapseConstants.ts

```diff
@@ -53,16 +53,16 @@ export const STUDY_ACTIVE: string = 'study active'
 export const STUDY_COMPLETE: string = 'study complete'
 export const DATABASE: string = 'database'
 export const PROGRAM: string = 'program'
-export const DATASET: string = 'dataset'
+export const DATASET = 'dataset'
```

Comment:
Removed the types so that TypeScript treats these as string literals, giving better type safety. Now when you try to specify the card type, it has to match one of the supported types.